style.css

```diff
@@ -47,16 +47,75 @@ header {
   height: 4rem;
   width: 100%;
   text-decoration: none;
+  align-items: center;
+}
+
+.menu {
+  display: none;
+}
+
+@media screen and (width <= 768px) {
+  .hamb-icon,
+  .close {
+    display: block;
+  }
+
+  .menu {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    min-width: 10rem;
+    gap: 4rem;
+    inset: 0 0 0 -50%;
+    background-color: #3c3a39;
+    padding: 2rem;
+    box-shadow: 0 0 0 100vmax #3c3a39;
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  .menu.visible {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .menu-desktop li {
+    min-width: 300%;
+    flex-direction: column;
+    align-items: end;
+    padding-bottom: 1rem;
+    margin-bottom: 2.5rem;
+    border-bottom: 1.5px grey solid;
+  }
+
+  .menu-desktop li a {
+    color: #fff5e1;
+    font-weight: 700;
+    font-size: 3rem;
+  }
```

Comment:
- [ ] Ouch! this is close but there seems to be some discrepancy in your design of the options link relative to the Figma design. Take a closer look at the font and also remove the underline from the options link to stay consistent with the Figma design, I'd also advise you to center the horizontal line. As a front-end developer, you have to pay attention to small details like this as they provide a polished UI. ✨
- Figma design:
  ![image](https://github.com/ulises2607/Portfolio/assets/104968236/67766481-9783-4bed-baa9-3f50d8af7673)
- Your design:
  ![image](https://github.com/ulises2607/Portfolio/assets/104968236/8f4aa2aa-65e9-4117-9422-ce6988ee5672)